### ISSUE TITLE
Manage and Source Public Bodies from Google Docs

### ISSUE THREAD

**Issue by tijptjik** [ODHK/opengov.hk#9]
- Format the Google [spreadsheet](https://docs.google.com/a/type.hk/spreadsheet/ccc?key=0Auo8nXUIP36xdEFSTFNaa2Y3TDR4TE5tNF9xeEwwLUE&usp=drive_web#gid=0) so it can be served up as JSON and imported to opengov.hk directly.
- Convert to the new Google Spreadsheet format

**Comment by 1000camels** [comment]
Let's find a way to publish and promote this spreadsheet as the defacto list for HKG. It is embarrassing that a proper Access to Information Officers list/directory (including email) does not exist. This an an opportunity to help get such a list created.

**Comment by tijptjik** [comment]
Definitely! Good point @1000camels . 

I'll also convert it to the latest GDocs format to futureproof it.

**Comment by tijptjik** [comment]
I've moved all the old spreadsheets with references to public bodies into the archive folder, and have created a new Docs Spreadsheet under the data folder as the canonical reference.

I also tested the import CSV functionality with the following link, and have successfully imported all the public bodies into production.

https://docs.google.com/spreadsheets/d/1x3WISjwPeC81n3GQTsqPxvKLnxpNehqyaWjdyCTX1bA/export?format=csv&id=1x3WISjwPeC81n3GQTsqPxvKLnxpNehqyaWjdyCTX1bA&gid=0

I tested the functionality locally and the records will be updated on the changes we make to the spreadsheet (as opposed to making new entities/ignoring them) as long as the name of the public body remains unchanged.

This should make it a cinch to update the records from time to time.
